tests/performance/log-results.js

```diff
@@ -87,7 +87,7 @@ const data = new TextEncoder().encode(
 		branch,
 		hash,
 		baseHash,
-		timestamp: parseInt( timestamp, 10 ),
+		date,
```

Comment:
The API still requires this to be named `timestamp`, see https://github.com/youknowriad/codevitals/blob/16542dcaf71f448fd0c29f19b18144284ce00106/pages/api/log.js#L12

```suggestion
		timestamp: date,
```

It then parses the string here: https://github.com/youknowriad/codevitals/blob/16542dcaf71f448fd0c29f19b18144284ce00106/pages/api/log.js#L36

Reply:
Ohh. Missed it completely